apps/trading/client-pages/markets/top-market-list.tsx

```diff
@@ -0,0 +1,51 @@
+import { EmblemByMarket } from '@vegaprotocol/emblem';
+import type { MarketMaybeWithCandles } from '@vegaprotocol/markets';
+import { useChainId } from '@vegaprotocol/wallet-react';
+import { Link } from 'react-router-dom';
+import {
+  priceValueFormatter,
+  priceChangeRenderer,
+  priceChangeSparklineRenderer,
+} from './use-column-defs';
+import { StackedCell } from '@vegaprotocol/datagrid';
+
+export const TopMarketList = ({
+  markets,
+}: {
+  markets?: MarketMaybeWithCandles[];
+}) => {
+  const { chainId } = useChainId();
+  return (
+    <>
+      {markets?.map((market) => {
+        return (
+          <div
+            className="grid auto-rows-min grid-cols-7 gap-3 text-xs"
+            key={market.id}
+          >
+            <span className="col-span-3 overflow-hidden">
+              <Link to={`/markets/${market.id}`}>
```

Comment:
Please use links object

```suggestion
              <Link to={Links.MARKET(market.id)}>
```